client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -181,17 +181,13 @@ public void testDecimal()
                         assertEquals(rs.getString(column), "9223372036854775774");
                     });
 
-            checkRepresentation(connectedStatement.getStatement(), "VARCHAR ''", Types.VARCHAR, (rs, column) -> {
```

Comment:
I find the previous representation easier to read than the current one because it is clearly delimited through the use of brackets.